apps/privacy-panel/js/app_list.js

```diff
@@ -102,18 +102,18 @@ define([], function() {
    *   .origin:      application origin
    */
   function _makeAppRepresentation(app) {
-    var manifest = app.manifest || app.updateManifest || {};
-
+    var manifest = new ManifestHelper(app.manifest ?
```

Comment:
Nit: the app.manifest || app.updateManifest form would work here too. No strong preference from me, it just doesnt need to change.